src/database/models/user.js

```diff
@@ -25,7 +25,7 @@ module.exports = (sequelize, DataTypes) => {
       role: { type: DataTypes.STRING, allowNull: false },
       password: {
         type: DataTypes.STRING,
-        allowNull: false,
+        allowNull: true,
```

Comment:
Why ?

Reply:
I didn't want the password to be required while creating the driver or operator